build/e2e-image/e2e.sh

```diff
@@ -13,9 +13,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+FEATURES=$1
+echo $FEATURES
 set -e
 echo "installing current release"
-DOCKER_RUN= make install
+DOCKER_RUN= make install FEATURE_GATES='"'$FEATURES'"'
 echo "starting e2e test"
-DOCKER_RUN= make test-e2e ARGS=-parallel=64
+DOCKER_RUN= make test-e2e ARGS=-parallel=64 FEATURE_GATES='"'$FEATURES'"'
```

Comment:
```suggestion
DOCKER_RUN= make test-e2e ARGS=-parallel=64 FEATURE_GATES=$FEATURES
```

Should be fine, no?

Reply:
This would not work because Features contains `&` which would be treated as new command. So the proposed version would work only with one feature flag.

Reply:
aaah I see. Interesting!